### PR TITLE
Added Limerol recipe.

### PR DESCRIPTION
**Pull request by bibliofilo**

### CODE REVIEW

**Comment by bibliofilo** [comment]
Sure thing, let me just find the original and rescale it. =)

File: src/recipes/limerol.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            "quantity": "15",
+            "measure": "ml",
+            "ingredient": "crushed ice"
+        }
+    ],
+    "directions": ["Clean the lime with hot water, dry and cut it into bits, then throw them into a long glass. Add sugar and mix well with the lemon juice.",
+        "Remove the watermelon pulp and cut it into cubes. Pour it into a blender with the Aperol and mix.",
+        "Fill the glass with 1/3 picked ice. Add the watermelon mix and stir well. Add the cava and serve with a straw."
+    ],
+    "image": "aviation.jpg",
```
Comment (alfg) [inline src/recipes/limerol.json]:
```suggestion
    "image": "limerol.jpg",
```

Also, the image file should be in `src/assets/recipes/`. Thanks!

**Comment by alfg** [comment]
Remove old image: `src/recipes/limerol.jpg`

**Comment by bibliofilo** [comment]
Done, and sorry, I thought I'd deleted it on my branch, but I must not have committed it properly.

**Comment by alfg** [comment]
Perfect! Thanks!

**Pull request merged**